zwave_js_server/model/notification.py

```diff
@@ -184,3 +184,47 @@ def status(self) -> PowerLevelTestStatus:
     def acknowledged_frames(self) -> int:
         """Return acknowledged frames property."""
         return self.data["args"]["acknowledgedFrames"]
+
+class MultilevelSwitchNotificationArgsDataType(TypedDict):
+    """Represent args for a Multi Level Switch CC notification event data dict type."""
+
+    event_type: int
+    direction: str
```

Comment:
```suggestion
class MultilevelSwitchNotificationArgsDataType(TypedDict, total=False):
    """Represent args for a Multi Level Switch CC notification event data dict type."""

    event_type: int  # required
    direction: str
```
I believe `direction` is optional here right?

Reply:
Yes, if the event type is a Multilevel Switch Stop event, then there is not direction associated with it.